css-fonts/styles.css

```diff
@@ -0,0 +1,8 @@
+p {
+  font-family: 'Merriweather', serif;
+  font-family: 'Dosis', sans-serif;
```

Comment:
you don't want to set two different font families on the same type of element, the second will override the first. I would apply one of these fonts to the `h1` and `h2` selectors